### ISSUE TITLE
Reduced use of typical gestures

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7325]
**Preferred term label:**
Reduced use of typical gestures

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal nonverbal communication

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
ORCID:0000-0002-4422-1708
